content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,170 @@
+- Model Registry [installed](/docs/components/model-registry/installation/)
+- Python >= 3.9, < 3.11
+
+## Example: track Model Artifacts from a Notebook
+
+This section details a step by step example on using Model Registry from a Notebook, installing and creating a client instance, indexing metadata, and retrieving metadata.
+
+### Install Model Registry Python client
+
+You can install the Model Registry python client in a Notebook, for instance with:
+
+```
+!pip install model-registry
+```
+
+Note: depending on your Python and Notebook environment, you might need to fine-tune the dependencies of: `ml-metadata`, `protobuf`, `grpcio`, or `tensorflow` if used.
+
+You can now create a client instance pointing to your deployed Model Registry from the previous steps.
+
+```python
+from model_registry import ModelRegistry
+
+registry = ModelRegistry(server_address="model-registry-service.kubeflow.svc.cluster.local", port=9090, author="your name")
+```
+
+You now have a Model Registry client instance: `registry`.
+
+### Register a Model Artifact metadata
+
+You can use the `register_model` method to index a model artifacts and its metadata, for instance:
+
+```python
+registeredmodel_name = "mnist"
+version_name = "v0.1"
+rm = registry.register_model(registeredmodel_name,
+                                "https://github.com/tarilabs/demo20231212/raw/main/v1.nb20231206162408/mnist.onnx",
+                                model_format_name="onnx",
+                                model_format_version="1",
+                                version=version_name,
+                                description="lorem ipsum mnist",
+                                metadata={
+                                    "accuracy": 3.14,
+                                    "license": "apache-2.0",
+                                }
+                                )
+```
+
+You can reference the pydoc documentation for additional information on indexing metadata on Model Registry, using the Model Registry Python client.
+
+### Retrieve a given Model Artifact metadata
+
+Continuing on the previous example, you can use the following methods to retrieve the metadata associated with a given Model Artifact:
+
+```python
+print("RegisteredModel:")
+print(registry.get_registered_model(registeredmodel_name))
+
+print("ModelVersion:")
+print(registry.get_model_version(registeredmodel_name, version_name))
+
+print("ModelArtifact:")
+print(registry.get_model_artifact(registeredmodel_name, version_name))
+```
+
+## Example add-on: deploy inference endpoint using Model Registry metadata 
+
+This section details a step by step example on using Model Registry to retrieve indexed ML artifacts metadata, and using that metadata to create an inference endpoint deployment.
+
+Note: the provided example uses the Model Registry Python client and KServe Python SDK. You can analogously make use of the Model Registry REST APIs, and your own Add-on SDK as needed.
```

Comment:
We might want to explain how this parameterization would be used to simplify new model deployment i.e. without the model registry, the user might need find metadata from several sources, which can make accurate correlation more difficult.  The model registry simplifies access to accurate metadata and enables you to automate deployment using the parameters based on model registry values.

Reply:
Worded as:

> Without Model Registry, you would need to fill this information manually and potentially from several sources, resulting in a not-trivial, manual process.
> Using Model Registry ensures simplified access to accurate metadata, and enables you to automate deployment based on the Model Registry values, as also shown in the example below.

in https://github.com/kubeflow/website/pull/3698/commits/f6cad0a1b55c9dcd8b1499f687cc20f3e2859671

thank you, kindly let me know if this matches your expectations.

Reply:
based [on event](https://github.com/kubeflow/website/pull/3698#issuecomment-2123542187) assume resolved, closing this thread. thank you